src/collector/collector.c

```diff
@@ -1597,6 +1597,7 @@ int main(int argc, char *argv[]) {
         glob->seqtrackers[i].intercepts = NULL;
         glob->seqtrackers[i].colident = &(glob->sharedinfo);
         glob->seqtrackers[i].encoding_method = glob->encoding_method;
+        glob->seqtrackers[i].enc_ber = wandder_init_encoder_ber(1000, 512);
```

Comment:
Is this something we should only be doing if `#ifdef HAVE_BER_ENCODING` applies?